### ISSUE TITLE
Try fails on unicode

### ISSUE THREAD

**Issue by jaredly** [reasonml/reasonml.github.io#214]
Apparently bucklescript is encoding unicode characters in a way that disagrees with json.
```
window.ocaml.compile('let x = {j|😤|j}')
vvv
 { "js_code" : "// Generated by BUCKLESCRIPT VERSION 1.9.2, PLEASE EDIT WITH CARE
'use strict';
var x = \"\240\159\152\164\";
exports.x = x;
/* No side effect */
" }"
```

The `\240` etc is not valid json, and so the try page fails with
![image](https://user-images.githubusercontent.com/112170/32105770-e452182e-bb29-11e7-9658-beadc91378ed.png)

**Comment by codekiln** [comment]
Related: #280

**Comment by chenglou** [comment]
Merged into #280. Thanks.
